### PR TITLE
Adds ping commands and some more Unicode emojis

### PR DESCRIPTION
**Pull request by RealKC**

### CODE REVIEW

File: FredBoat/src/main/java/fredboat/command/maintenance/PingCommand.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    }
+    
+    @Override
+    public void onInvoke(Guild guild, TextChannel channel, Member invoker, Message message, String[] args) {
+
+        JDA jda = guild.getJDA();
+        long ping = jda.getPing();
+        String pingMilliSeconds = String.valueOf(ping) + "ms";
+
+        MessageEmbed embed = new EmbedBuilder()
```
Comment (Frederikam) [inline FredBoat/src/main/java/fredboat/command/maintenance/PingCommand.java]: I feel the embed is unnecessary and inconsistent with the rest of the bot. IMO embeds should only be used when normal messages are insufficient.

Comment (RealKC) [inline FredBoat/src/main/java/fredboat/command/maintenance/PingCommand.java]: ok

**Comment by RealKC** [comment]
This should be fine now?

**Comment by RealKC** [comment]
Wait, I want to add one more alias before merging

File: FredBoat/src/main/java/fredboat/command/maintenance/PingCommand.java [hunk 2]
```diff
@@ -24,15 +24,8 @@ public void onInvoke(Guild guild, TextChannel channel, Member invoker, Message m
...
-        String pingMilliSeconds = String.valueOf(ping) + "ms";
 
-        MessageEmbed embed = new EmbedBuilder()
-                .setDescription(pingMilliSeconds)
-                .setColor(FREDBOAT_COLOR)
-                .setFooter(channel.getJDA().getSelfUser().getName(), channel.getJDA().getSelfUser().getAvatarUrl())
-                .build();
-
-        channel.sendMessage(embed).queue();
+        channel.sendMessage(String.valueOf(ping) + "ms";).queue();
```
Comment (Frederikam) [inline FredBoat/src/main/java/fredboat/command/maintenance/PingCommand.java]: `String.valueOf()` in unnecessary here. Java will automatically translate the number to characters

**Comment by Frederikam** [comment]
>[ERROR] /home/travis/build/Frederikam/FredBoat/FredBoat/src/main/java/fredboat/command/maintenance/PingCommand.java:[28,40] ')' expected
[ERROR] /home/travis/build/Frederikam/FredBoat/FredBoat/src/main/java/fredboat/command/maintenance/PingCommand.java:[28,41] illegal start of expression
[ERROR] /home/travis/build/Frederikam/FredBoat/FredBoat/src/main/java/fredboat/command/maintenance/PingCommand.java:[28,42] illegal start of expression
[ERROR] /home/travis/build/Frederikam/FredBoat/FredBoat/src/main/java/fredboat/command/maintenance/PingCommand.java:[28,48] ';' expected

**Comment by RealKC** [comment]
Yes I saw that

**Comment by RealKC** [comment]
It failed again?!
(When clicking on Details I see this
>[22:13:37] [ WARN ] [Config] Could not find file './credentials.yaml', looking for legacy './credentials.json' to rewrite
[22:13:37] [ ERROR] [Config] No credentials file is present. Bot cannot run without it. Check the documentation.
[22:13:37] [ INFO ] [ProvideJDASingleton] Credentials and/or config files not found, live tests won't be available
Exception in thread "RandomImageCommand imgur updater" java.lang.NullPointerException
	at fredboat.command.fun.RandomImageCommand.populateItems(RandomImageCommand.java:123)
	at fredboat.command.fun.RandomImageCommand.lambda$new$5(RandomImageCommand.java:73)
	at fredboat.command.fun.RandomImageCommand$$Lambda$140/1205406622.run(Unknown Source)
	at java.lang.Thread.run(Thread.java:745)
Exception in thread "RandomImageCommand imgur updater" java.lang.NullPointerException
	at fredboat.command.fun.RandomImageCommand.populateItems(RandomImageCommand.java:123)
	at fredboat.command.fun.RandomImageCommand.lambda$new$5(RandomImageCommand.java:73)
	at fredboat.command.fun.RandomImageCommand$$Lambda$140/1205406622.run(Unknown Source)
	at java.lang.Thread.run(Thread.java:745)
Exception in thread "RandomImageCommand imgur updater" java.lang.NullPointerException
	at fredboat.command.fun.RandomImageCommand.populateItems(RandomImageCommand.java:123)
	at fredboat.command.fun.RandomImageCommand.lambda$new$5(RandomImageCommand.java:73)
	at fredboat.command.fun.RandomImageCommand$$Lambda$140/1205406622.run(Unknown Source)
	at java.lang.Thread.run(Thread.java:745)
Exception in thread "RandomImageCommand imgur updater" java.lang.NullPointerException
	at fredboat.command.fun.RandomImageCommand.populateItems(RandomImageCommand.java:123)
	at fredboat.command.fun.RandomImageCommand.lambda$new$5(RandomImageCommand.java:73)
	at fredboat.command.fun.RandomImageCommand$$Lambda$140/1205406622.run(Unknown Source)
	at java.lang.Thread.run(Thread.java:745)
Exception in thread "RandomImageCommand imgur updater" java.lang.NullPointerException
	at fredboat.command.fun.RandomImageCommand.populateItems(RandomImageCommand.java:123)
	at fredboat.command.fun.RandomImageCommand.lambda$new$5(RandomImageCommand.java:73)
	at fredboat.command.fun.RandomImageCommand$$Lambda$140/1205406622.run(Unknown Source)
	at java.lang.Thread.run(Thread.java:745)
Exception in thread "RandomImageCommand imgur updater" java.lang.NullPointerException
	at fredboat.command.fun.RandomImageCommand.populateItems(RandomImageCommand.java:123)
	at fredboat.command.fun.RandomImageCommand.lambda$new$5(RandomImageCommand.java:73)
	at fredboat.command.fun.RandomImageCommand$$Lambda$140/1205406622.run(Unknown Source)
	at java.lang.Thread.run(Thread.java:745)

and I didn't mess with any of that stuff)

**Comment by napstr** [comment]
Those exceptions don't fail the tests.

This is the actual reason:

```
Tests run: 1, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 3.655 sec <<< FAILURE! - in fredboat.commandmeta.CommandInitializerTest
testHelpStrings()  Time elapsed: 2.675 sec  <<< FAILURE!
org.opentest4j.AssertionFailedError: fredboat.command.maintenance.PingCommand.help() returns null ==> expected: not <null>
	at fredboat.commandmeta.CommandInitializerTest.testHelpStrings(CommandInitializerTest.java:37)
```

Commands are required to provide a help text.

**Comment by RealKC** [comment]
>Commands are required to provide a help text.

Why won't you let me be lazy? :p Anyway, I'll do the help text and have a //TODO: i18n. (i18n was internalisation, right?)

**Comment by RealKC** [comment]
YES, Travis thinks it's fine! Time to merge?

File: FredBoat/src/main/java/fredboat/command/maintenance/PingCommand.java [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+        channel.sendMessage(ping + "ms").queue();
+    }
+}
+
+//hello
+//this is a comment
+//I want pats
+//multiple pats
+//pats never seen before
```
Comment (Shredder121) [inline FredBoat/src/main/java/fredboat/command/maintenance/PingCommand.java]: 👀

File: FredBoat/src/main/java/fredboat/commandmeta/init/MainCommandInitializer.java [hunk 4]
```diff
@@ -101,13 +102,15 @@ public static void initCommands() {
         CommandRegistry.registerCommand("faceofdisapproval", new TextCommand("ಠ_ಠ"), "fod", "disapproving");
         CommandRegistry.registerCommand("sendenergy", new TextCommand("༼ つ ◕_◕ ༽つ"));
         CommandRegistry.registerCommand("peeking", new TextCommand("┬┴┬┴┤ ͜ʖ ͡°) ├┬┴┬┴"), "peekinglenny", "peek");
-        CommandRegistry.registerCommand("dealwithit", new TextCommand("(•\\_•) ( •\\_•)>⌐■-■ (⌐■_■)"), "dwi"); //NOTE: This may break, depending on how the backlashes are handled.
+        CommandRegistry.registerCommand("dealwithit", new TextCommand("(•\\_•) ( •\\_•)>⌐■-■ (⌐■_■)"), "dwi"); //NOTE: This may break, depending on how the backlashes are handled. UPDATE: This works fine
```
Comment (Shredder121) [inline FredBoat/src/main/java/fredboat/commandmeta/init/MainCommandInitializer.java]: Just remove the `NOTE:` then :P

File: FredBoat/src/main/java/fredboat/commandmeta/init/MainCommandInitializer.java [hunk 5]
```diff
@@ -101,13 +102,15 @@ public static void initCommands() {
...
         CommandRegistry.registerCommand("channelingenergy", new TextCommand("(ﾉ◕ヮ◕)ﾉ*:･ﾟ✧ ✧ﾟ･: *ヽ(◕ヮ◕ヽ)"));
         CommandRegistry.registerCommand("butterfly", new TextCommand("Ƹ̵̡Ӝ̵̨̄Ʒ"));
-        CommandRegistry.registerCommand("angrytableflip", new TextCommand("(ノಠ益ಠ)ノ彡┻━┻"), "tableflipbutangry");
+        CommandRegistry.registerCommand("angrytableflip", new TextCommand("(ノಠ益ಠ)ノ彡┻━┻"), "tableflipbutangry", "atp");
         CommandRegistry.registerCommand("cooldog", new DogCommand(), "dog", "dogmeme");
-        CommandRegistry.registerCommand("lood", new TextCommand("T-that's l-lewd, baka!!!"), "lewd");
+        CommandRegistry.registerCommand("lood", new TextCommand("T-that's l-lewd, baka!!!"), "lewd", "l00d");
         CommandRegistry.registerCommand("useless", new TextCommand("This command is useless."));
+        CommandRegistry.registerCommand("swtf", new TextCommand("¯\\(°_o)/¯"), "shrugwtf");
+        CommandRegistry.registerCommand("hurray", new TextCommand("ヽ(^o^)ノ"));
```
Comment (Shredder121) [inline FredBoat/src/main/java/fredboat/commandmeta/init/MainCommandInitializer.java]: *gasp* can we have an alias `yay`?

Comment (RealKC) [inline FredBoat/src/main/java/fredboat/commandmeta/init/MainCommandInitializer.java]: `woot` as well?

File: FredBoat/src/main/java/fredboat/command/maintenance/PingCommand.java (hunk 3 above)
Comment (RealKC) [inline FredBoat/src/main/java/fredboat/command/maintenance/PingCommand.java]: Pats are good, that is undeniable

Comment (Shredder121) [inline FredBoat/src/main/java/fredboat/command/maintenance/PingCommand.java]: but not necessary in source code? :⭕️

Comment (RealKC) [inline FredBoat/src/main/java/fredboat/command/maintenance/PingCommand.java]: Easter eggs (ノಠ益ಠ)ノ彡┻━┻

File: FredBoat/src/main/java/fredboat/commandmeta/init/MainCommandInitializer.java (hunk 5 above)
Comment (RealKC) [inline FredBoat/src/main/java/fredboat/commandmeta/init/MainCommandInitializer.java]: Wot about `woo`?

File: FredBoat/src/main/java/fredboat/commandmeta/init/MainCommandInitializer.java (hunk 4 above)
Comment (RealKC) [inline FredBoat/src/main/java/fredboat/commandmeta/init/MainCommandInitializer.java]: Gonna do, RIP //NOTE: 2017-2017

File: FredBoat/src/main/java/fredboat/commandmeta/init/MainCommandInitializer.java (hunk 5 above)
Comment (Shredder121) [inline FredBoat/src/main/java/fredboat/commandmeta/init/MainCommandInitializer.java]: Meh, we could, but I think too many (close) aliases should be avoided?

Comment (RealKC) [inline FredBoat/src/main/java/fredboat/commandmeta/init/MainCommandInitializer.java]: Guess `hurray`, `woot`, and `yay` are enough, we shall summon Fred after this, ja?

Comment (RealKC) [inline FredBoat/src/main/java/fredboat/commandmeta/init/MainCommandInitializer.java]: Yeah that's true

**Comment by RealKC** [comment]
Aye aye Capt'n @Frederikam, time add this plank to the boat!


*Pls merge, kthx.*

**Pull request merged**